source/common/upstream/cluster_manager_impl.cc

```diff
@@ -876,16 +884,15 @@ ClusterManagerImpl::ThreadLocalClusterManagerImpl::ClusterEntry::httpConnPool(
   // Select a host and create a connection pool for it if it does not already exist.
   auto ret = connPool(priority, protocol, context, false);
 
-  // Now see if another host should be prefetched.
-  // httpConnPool is called immediately before a call for newStream. newStream doesn't
-  // have the load balancer context needed to make selection decisions so prefetching must be
+  // Now see if another host should be preconnected.
+  // httpConnPoolForCluster is called immediately before a call for newStream. newStream doesn't
```

Comment:
(merge issue, feel free to fix in a follow up)

```suggestion
  // httpConnPool is called immediately before a call for newStream. newStream doesn't
```